paper_on_queues_and_stack_structures.md

```diff
@@ -0,0 +1,21 @@
+# Introduction to Queues and Stack Structures  
+Queues and stack are data structures that differ when it comes to the modification of data. The principle for queues is the data that is first in, gets out first. For stacks, the principle is last in, first out. 
+A queue can be visualized as a sequence of rectangular boxes arranged from left to right with random objects (data) in each one.  With a stack, the same rectangle boxes are arranged bottom to top similar to a stack of paper. Queues and stack structures have their own trade-offs and they both play vital roles in the computer science.
+
+### Queues
+Queues operate on the principle that the first data to be inserted is the first data to be deleted.  Data is inserted at the end of the list and deleted at the front of the list. You can see this principle applied bank tellers service customers that enter the customer service line -- customers service in the order they entered the line.  Queues advantages or disadvantages, depending on the use. For example, queues are advantageous when managing text messages you send a friend. Additionally, data elements have to be deleted in order for new data to be added. The insertion and deletion of data in the middle of a queue can also be quite time consuming. Queues are excellent at regulating priorities such as with regards to a traffic flows. Printer queues and messaging are also common uses of queues that take advantage of its first come first serve principle.
+
+### Stacks
+However, stack structures have slightly differing advantages and disadvantages against queues because of the nature of stacks. Stacks are great at backtracking and reviewing historical events as the data on the top can just be deleted or reviewed. The access of more buried data in a stack structure can also prove to be tedious, as data at the middle or bottom of the stack can only be accessed if the data above it is removed. Again stacks are great if you are trying to review historical information.
```

Comment:
Delete "However," Stack should be the first word
Delete "slightly"
Delete "because of the nature of stacks"
What does historical events mean in this context? I'm not understanding why stacks are good for reviewing this. This needs an example.
Delete "just"
Delete the last sentence.